suitcase/json_metadata/tests.py

```diff
@@ -1,2 +1,70 @@
-# Tests should generate (and then clean up) any files they need for testing. No
-# binary files should be included in the repository.
+from collections import defaultdict
+import event_model
+from . import export
+import json
+
+
+def create_expected(collector):
+    '''This collects the metadata into a dict to compare to the loaded data
+    '''
+
+    def _tuples_to_lists(val):
+        ''' Recursively convert tuples to lists'''
+        if type(val) in [tuple, list]:
+            items = []
+            for item in val:
+                items.append(_tuples_to_lists(item))
+            val = items
+        return val
+
+    expected = {'metadata': {}}
+    expected['metadata']['descriptors'] = defaultdict(dict)
+    for name, doc in collector:
+        if name in ['start', 'stop']:
+            sanitized_doc = _apply_to_dict_recursively(doc, _tuples_to_lists)
```

Comment:
Any reason not to use `event_model.sanitize_doc` here? It may or may not be overkill, but at least it's simple.

Reply:
sanitize doc does not replace tuples with lists (at least when I tested it). The issue here is not that there are any numpy arrays (which is what `event_model.sanitize_docs` resolves) but that we have tuples in this version however the loaded data file has replaced those tuples with lists. So to do the `assert_array_equal` I need to convert all of the tuples to lists. 

Initially I had used `event_model._apply_to_dict_recursively` but that has since been deprecated.

Reply:
> sanitize doc does not replace tuples with lists

I ran into the same problem, which was part of the motivation for https://github.com/NSLS-II/event-model/pull/43. As of 1.8.0rc2, `sanitize_doc` replaces tuples with lists.

Reply:
ok I will make that change as well !-)